packages/tokens/package.json

```diff
@@ -0,0 +1,19 @@
+
+  {
+    "name": "@leafygreen-ui/tokens",
+    "version": "0.9.0",
```

Comment:
I'm wondering if we release this as a beta version.

I feel pretty confident that we're going to want to change this API as we learn more. And the best way for us to learn more is for us to start consuming these values in other leafygreen components.

I think we should see how our usages change the APIs in here. I'd feel more comfortable giving ourselves a pre-release version so we can have space to iterate (in public) as we refine our usages.

Reply:
I'm fine with that. I felt most strongly about getting this out the door as a utility for .design but definitely agree that we'll know more about the desired API as we use it ourselves. We don't have to announce this package externally until we feel more confident

Reply:
Let's change this version in that case, as long as we get agreement from everyone! @DesignerDave

Reply:
SGTM. So are we leaving the API as is for now?